src/app.js

```diff
@@ -197,11 +199,14 @@ export function app(app) {
     function removeChild() {
       parent.removeChild(element)
     }
+    recoverNode(node)
   }
 
   function patch(parent, element, oldNode, node) {
     if (oldNode == null) {
       element = parent.insertBefore(createElement(node), element)
+    } else if (typeof node === "string" && typeof oldNode === "string") {
```

Comment:
Well, not recycling exactly but it's related.  Since we don't ever update textNodes currently GC is busy with collecting them, since recycling is about reducing GC I thought why not try this also.